src/PrestaShopBundle/Resources/config/admin/services-form.yml

```diff
@@ -224,6 +224,14 @@ services:
         tags:
             - { name: form.type }
 
+    form.type.maintenance.general:
+        class: 'PrestaShopBundle\Form\Admin\ShopParameters\General\MaintenanceType'
+        parent: 'form.type.translatable.aware'
+        calls:
+            - [ setLocales, ["@=service('prestashop.adapter.legacy.context').getLanguages()"]]
```

Comment:
@mickaelandrieu, don't you think we should move that call in the service `form.type.translatable.aware`?